### DISCUSSION TITLE
Disable a node by its node ID

### DISCUSSION THREAD

**Discussion by SciManJD** [bcakmakoglu/vue-flow#946] [Q&A] [answered]
Is there a way to disable a node by its ID? I want to avoid sending of events and the style of the node. It is similar what we can do if we disable for example a standard input element. It becomes gray and can't write anything in it.

**Comment by bcakmakoglu** [comment, marked as answer]
You can turn off dragging, selecting, focusing, deleting etc. - but "disabling" a node is not a built-in functionality, no.
[You can find the full properties list of a node here.](https://vueflow.dev/typedocs/interfaces/Node.html)

Styling is up to you, the library won't do it for you.

**Reply by SciManJD** [reply]
Ok, I can control the 'selectable' property from my code. I also want to change the style, but I can find the right way to do it. I have the following code:

```
                        let node = this.instance.findNode(nodeId);
                        // Node found?
                        if(node) {
                            // Disable it
                            node.selectable = false;
                            node.styles = ????
                         }
```
I couldn't find a suitable example how to set the style. Can you give me a link, please? I want to change the opacity for example. Can I do this?

**Reply by SciManJD** [reply]
It was easy:

`node.style = {opacity: '0.5'};`
